services/web/client/source/class/qxapp/ui/markdown/Markdown.js

```diff
@@ -22,6 +22,20 @@ qx.Class.define("qxapp.ui.markdown.Markdown", {
   construct: function(markdown) {
     this.base(arguments);
     this.setRich(true);
+    this.__loadMarked = new Promise((resolve, reject) => {
+      if (typeof marked === "function") {
+        resolve(marked);
+      } else {
+        const loader = new qx.util.DynamicScriptLoader("marked/marked.min.js");
+        loader.addListenerOnce("ready", () => {
+          resolve(marked);
+        }, this);
+        loader.addListenerOnce("failed", e => {
+          reject(Error(`Failed to load ${e.getData()}. Value couldn't be updated.`));
```

Comment:
How does this display to the final user? Shouldn't we have some kind of double tab (as in GitHub) so the user can preview our markdown?

Reply:
Is there any kind of protection against injecting malicious html/code on the markdown?

Reply:
> Is there any kind of protection against injecting malicious html/code on the markdown?

I'll add the [DOMPurify](https://github.com/cure53/DOMPurify)

Reply:
@ignapas I think we can close this PR since this content was already merged in #1019